python/tests/unit/utils/agent_diagnostics/test_trace_open_ai_assistant_agent.py

```diff
@@ -0,0 +1,33 @@
+# Copyright (c) Microsoft. All rights reserved.
+
+
+from unittest.mock import patch
+
+import pytest
+
+from semantic_kernel.agents.open_ai.open_ai_assistant_agent import OpenAIAssistantAgent
+from semantic_kernel.exceptions.agent_exceptions import AgentInitializationException
+
+
+@patch("semantic_kernel.utils.telemetry.agent_diagnostics.decorators.tracer")
+async def test_open_ai_assistant_agent_invoke(mock_tracer, chat_history):
```

Comment:
The failure is getting raised from the `open_ai_assistant_agent = OpenAIAssistantAgent()` call. Same with the streaming test below.